price_parser/parser.py

```diff
@@ -142,7 +162,9 @@ def extract_currency_symbol(price: Optional[str],
     for meth, attr in methods:
         m = meth(attr) if attr else None
         if m:
-            return m.group(0)
+            groups = [match for match in m.groups() if match is not None]
+            if groups:
```

Comment:
Acording to the coverage data, this always evaulates to `True` with the current tests. It would be great to add a test for the case where it evaluates to `False` or, if that cannot happen, replace the if by an `assert`.